### PR TITLE
Add conda install and release instructions

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
- Add instructions for installation using conda
- Update release instructions

### CODE REVIEW

**Comment by thehomebrewnerd** [comment]
I tested the install instructions using conda for both nlp-primitives and nlp-primitives-complete, and they work as expected.

When instantiating the UniversalSentenceEncoder with the complete install, several future warnings are generated similar to this: `/Users/nate.parsons/miniconda3/envs/nlp-complete/lib/python3.7/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.`, but it works as expected.

File: README.md [hunk 1]
```diff
@@ -7,17 +7,29 @@ nlp_primitives is a Python library with Natural Language Processing Primitives,
...
+There are two options for installing nlp_primitives. Both of the options will also install Featuretools if it is not already installed.
 
-The first option is to install a version of nlp_primitives that does not include Tensorflow. With this option, primitives that depend on Tensorflow cannot be used. Currently, the only primitive that can not be used with this install option is ``UniversalSentenceEncoder``. To install nlp_primitives without Tensorflow run:
+The first option is to install a version of nlp_primitives that does not include Tensorflow. With this option, primitives that depend on Tensorflow cannot be used. Currently, the only primitive that can not be used with this install option is ``UniversalSentenceEncoder``.
+
+nlp_primitives without Tensorflow can be installed with pip:
 ```shell
 pip install nlp_primitives
 ```
+or from the conda-forge channel:
```
Comment (rwedge) [inline README.md:18]: maybe "or from the conda-forge channel on conda"

File: README.md [hunk 2]
```diff
@@ -7,17 +7,29 @@ nlp_primitives is a Python library with Natural Language Processing Primitives,
...
+```
 
-The second option is to install the complete version of nlp_primitives, which will also install Tensorflow and allow use of all primitives. To install the complete version of nlp_primitives:
+The second option is to install the complete version of nlp_primitives, which will also install Tensorflow and allow use of all primitives. 
+
+To install the complete version of nlp_primitives with pip:
 ```shell
 pip install "nlp_primitives[complete]"
 ```
+or from the conda-forge channel:
```
Comment (rwedge) [inline README.md:29]: same idea here with mentioning conda explicitly

File: release.md [hunk 3]
```diff
@@ -2,19 +2,132 @@
...
+* Release description should be the full changelog updates for the release, including the line thanking contributors. Contributors should also have their links changed from the docs syntax (:user:`rwedge`) to github syntax (@rwedge)
+* This is not a pre-release
+* Publishing the release will automatically upload the package to PyPI
 
-## Release on PyPI
-After publishing the release on GitHub, a GitHub Action will automatically upload the package to PyPI.
\ No newline at end of file
+## Release on conda-forge
+1. A bot should automatically create a new PR in conda-forge/nlp_primitives-feedstock
+2. Update requirements changes in `recipe/meta.yaml` (bot should have handled version and source links on its own)
```
Comment (rwedge) [inline release.md:132]: Non-maintainers will need a maintainer to merge their changes into the bot's pr

File: README.md (hunk 2 above)
Comment (thehomebrewnerd) [reply README.md:29]: Updated

File: README.md (hunk 1 above)
Comment (thehomebrewnerd) [reply README.md:18]: Updated

File: release.md (hunk 3 above)
Comment (thehomebrewnerd) [reply release.md:132]: Added

**Pull request merged**